### PR TITLE
(localize) distil strings from wallet Send and receive pages

### PR DESCRIPTION
**Pull request by gasolin**
related to #438

- Update readme
- distil strings from wallet Send and receive pages (includes the protocol swither)

截圖 2022-06-01 下午11 19 13

TODO
- will deal SharedAssetInput component when localize the swap UI
- Acitivities are not translated yet since some code condition depends on the label name
https://github.com/tallycash/extension/blob/main/ui/pages/SignTransaction.tsx#L82

### CODE REVIEW

File: ui/_locales/en/messages.json [hunk 1]
```diff
@@ -31,10 +27,70 @@
...
+    "description": "in TopMenu > TopMenuProtocolList"
+  },
+  "ProtocolCompatibleChain": {
+    "message": "Ethereum-compatible blockchain",
+    "description": "in TopMenu > TopMenuProtocolList"
+  },
+  "ProtocolConnected": {
+    "message": "Connected",
+    "description": "in TopMenu > TopMenuProtocolListItem"
+  },
```
Comment (jagodarybacka) [inline ui/_locales/en/messages.json:45]: let's start with a lowercase letters so `"protocol___": { ...`

**Review by jagodarybacka — CHANGES_REQUESTED** [review]
Looks good, just one minor thing:

File: ui/_locales/en/messages.json (hunk 1 above)
Comment (gasolin) [reply ui/_locales/en/messages.json]: thanks, comment addressed

**Review by jagodarybacka — APPROVED** [review]
Works fine, let's 🚀

**Pull request merged**